graph_train.py

```diff
@@ -0,0 +1,178 @@
+import os
+import argparse
+import torch
+from collections import defaultdict
+import random
+
+import nltk
+import torch_geometric.data as geom_data
+# nltk.download('reuters')
+# from nltk.corpus import reuters
+from datasets.reuters_graph_datasets import R8, R52
+from torch_geometric.data import Data
+import torch.nn.functional as F
+from torch_geometric.nn import GCNConv
+import pytorch_lightning as pl
+from pytorch_lightning.callbacks import ModelCheckpoint
+
+
+from datasets.graph_utils import PMI, tf_idf_mtx
+
+device = torch.device("cuda:0") if torch.cuda.is_available() else torch.device("cpu")
+
+class Net(torch.nn.Module):
+    def __init__(self, num_nodes):
+        super(Net, self).__init__()
+        self.conv1 = GCNConv(num_nodes, 200)
+        self.conv2 = GCNConv(200, 8)
+
+    def forward(self, data):
+        x, edge_index, edge_weight = data.x, data.edge_index, data.edge_attr
+        x = self.conv1(x, edge_index, edge_weight)
+        x = F.relu(x)
+        x = F.dropout(x, training=self.training)
+        x = self.conv2(x, edge_index, edge_weight)
+        return x
+
+class Graph_model(pl.LightningModule):
```

Comment:
Isn't this also in model.py?

Reply:
Yes this file should also be deleted in the end. Maybe for this file, we can already do it right now as we can always track it back if we need to. do you agree on deleting?

Reply:
Hm it's fine for now I guess. Maybe next time when we do this we could add a comment on top of the file like "THIS FILE IS A DUPLICATE AND WILL BE DELETED" or something. Also to not make our future code depend on it if someone does not know our plans with the file

Reply:
haha good idea. I added it